nextflow.config

```diff
@@ -220,6 +221,12 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    gpu {
+        docker.runOptions       = '-u $(id -u):$(id -g) --gpus all'
+        apptainer.runOptions    = '--nv'
+        singularity.runOptions  = '--nv'
+        use_gpu                 = true
```

Comment:
Is this variable being used at all anywhere?

Reply:
which variable ? `use_gpu` ?

Reply:
it is being used for the bwameth gpu test

Reply:
Yes, `use_gpu `.

Reply:
That's `params.use_gpu` defined here,

https://github.com/nf-core/methylseq/blob/8c6210663a9189df0f2ea3c7c522a3efac5b4510/nextflow.config#L78

But the `use_gpu` variable inside the `gpu` profile is unnecessary. It can be confusing.

Reply:
I think we are doing that for consistency across pipelines. will be added to template in future.

Reply:
ur right...not needed here... I will remove it in next PR before release